beetsplug/discogs.py

```diff
@@ -425,6 +425,8 @@ def get_album_info(self, result):
             catalogno = result.data["labels"][0].get("catno")
             labelid = result.data["labels"][0].get("id")
 
+        cover_art_url = self.select_cover_art(result)
```

Comment:
Maybe we only update the value if the function does not return `None`.

Reply:
Sorry @arsaboo, I'm not sure what you mean. Do you mean we should not pass `cover_art_url` to the `AlbumInfo` constructor at all if its value is `None`?